src/agentlab/experiments/launch_exp.py

```diff
@@ -20,6 +20,7 @@ def run_experiments(
     exp_args_list: list[ExpArgs],
     study_dir,
     parallel_backend="joblib",
+    save_transition_history=False,
```

Comment:
saving transition history should be done by placing them inside the AgentInfo. I -and @ThibaultLSDC - can help you figure this out :)